src/components/HeaderForm.js

```diff
@@ -752,6 +762,9 @@ class HeaderForm extends Component {
     const examplesFlag = this.state.dataType === dataTypes.EXAMPLES;
     const viewTargetHasChange = !viewTargetsEqual(this.getNextViewTarget(), this.props.getCurrentViewTarget());
 
+    const ifReadsExist = readsExist(this.state.tracks);
```

Comment:
It looks like this variable is sued exactly once, so maybe the function call should just be made where the variable is used?

Or if you need the variable to simplify the complicated logic where it is used, maybe we could move the variable closer to the logic it goes with?